README.md

```diff
@@ -189,10 +189,71 @@ For more documentation about PSPDFKit check out [PSPDFKit online documentation](
 
 Xamarin.Forms does not allow highjacking the `ContentPage`, so by default only fullscreen presentation is allowed. With the new [Xamarin.Forms embedding](https://blog.xamarin.com/unleashed-embedding-xamarin-forms-in-xamarin-native/), you can take any ContentPage and add it to your native applications.
 
+# PSPDFKit Instant
+
+Support for Instant was added with the Xamarin.Android Bindings for PSPDFKit 4.5.1 for Android.
+
+With PSPDFKit Instant, it’s easier than ever to add real-time collaboration features to your PSPDFKit-powered app, allowing your users to seamlessly share, edit, and annotate PDF documents across iOS, Android, and web. With just a few lines of code, PSPDFKit Instant gives your users a massive productivity boost.
+
+For more information about Instant, please have a look at our [website](https://pspdfkit.com/instant/).
+
+# Examples
+
+You can find several sample projects in the `samples` folder, including a catalog and Xamarin.Forms example.
+
+## How to run the example projects
+
+1. Complete **Step 1** to download additional resources needed for the examples.
+2. Open the `PSPDFKit.Android.sln` solution in Visual Studio.
+3. Select the example project and device you want to run it on (alternatively you can also right-click on the project and select "Build `Project Name`").
+<img width="60%" src="Images/Project-setup.png"/>
+4. Tap on the triangle on the left to run the project.
+
+### PSPDFKit Instant Example
+
+This example is included in the PSPDFCatalog example, but you can also find the code [here][Instant Example].
+
+The PSPDFKit Instant example shows how easy and efficient Instant works. Just go the [Instant demo page](https://pspdfkit.com/instant/demo/) and tap on `Instant on Android`, this will show a code at step three, which you have to enter in the example on your device. Afterwards you'll be connected to the server and you can start testing!
+
+<div id="image-table">
+     <table>
+  	    <tr>
+      	    <td>
+             <img width="90%" src="Images/Instant-device.png"/>
+           </td>
+           <td>
+             <img width="90%" src="Images/Instant-desktop.png"/>
+           </td>
+       </tr>
+    </table>
+</div>
+
+
+### PSPDFCatalog
+
+The `PSPSDFCatalog` project has various little example projects in it, including a basic example, which opens a demo document in PSPSDFKit, an example for automatically filling out forms, or one for creating a password protected PDF.
+
+<img width="80%" src="Images/Catalog.png"/>
+
+
+### Xamarin.Forms
+
+`XFSample.Droid` is an example showcasing how to build an app with PSPDFKit using Xamarin.Forms.
+
+<img width="80%" src="Images/XForms.png"/>
+
+
+### AndroidSimple
+
+`AndroidSimple` is an example project showing showing how to open a preset document, or how to open a document from various file providers or the local file directory.
```

Comment:
"to open a preset document" -> "to open **and** preset document"

Reply:
I misread "preset" for "present". 🤦‍♂️ 

In this context, I'd go with "to open and present a document"